loadbalancer.tf

```diff
@@ -68,6 +68,7 @@ resource "aws_lb_target_group" "keycloak-target-group" {
     matcher             = "200"
     timeout             = "3"
     path                = "/auth/health"
```

Comment:
@jurakp is the path still the same even though the port has changed?

Reply:
@ianwestcott yes, path is the same - only port has been changed. This is working well in sandbox, should be also in dev.

Reply:
@ianwestcott Yes, the have moved the path to the different port. originally it was on 8080. I hope they do not want to change again in the future :)